typed-racket-more/typed/net/url.rkt

```diff
@@ -1,86 +1,70 @@
 #lang typed/racket/base
 
-(require typed/private/utils)
-
-(require-typed-struct/provide path/param ([path : (U String 'up 'same)] [param : (Listof String)]) net/url)
-
-(require-typed-struct/provide
- url ([scheme : (Option String)]
-      [user : (Option String)]
-      [host : (Option String)]
-      [port : (Option Integer)]
-      [path-absolute? : Boolean]
-      [path : (Listof path/param)]
-      [query : (Listof (Pair Symbol (Option String)))]
-      [fragment : (Option String)])
- net/url)
-
-(require/opaque-type URL-Exception url-exception? net/url)
-(provide URL-Exception url-exception?)
-
-(require/opaque-type HTTP-Connection http-connection? net/url)
-(provide HTTP-Connection http-connection?)
-
-(define-type-alias PortT (case-lambda (url -> Input-Port) (url (Listof String)-> Input-Port)))
```

Comment:
You can use `:type` for this:

```
> (define-type S (-> Integer Integer))
> (:type S)
(-> Integer Integer)
[can expand further: Integer]
```

But the larger issue is that we can't just drop exported identifiers, since that may break people's programs.